### PR TITLE
feat: allow disabling of loader cache

### PR DESCRIPTION
**Pull request by thebigredgeek**

### CODE REVIEW

**Comment by thebigredgeek** [comment]
Bump?  Not sure what is going on with codecov above, but I am anxious to get this change into the mainline so that I can stop consuming my fork in production haha

**Comment by geekuillaume** [comment]
Bump ?

**Comment by thebigredgeek** [comment]
bump?

**Comment by bebraw** [comment]
Can you document `disableCaching`? Test would be good to add too.

**Comment by thebigredgeek** [comment]
@bebraw is this issue still active?  If so, I'd be happy to update this with tests and docs

**Comment by scottdj92** [comment]
Please add tests and docs :)

**Comment by michael-ciniawsky** [comment]
@thebigredgeek Please close and reopen the PR to trigger the CLA bot again and sign the CLA :)

File: loader.js [hunk 1]
```diff
@@ -122,3 +126,6 @@ module.exports.pitch = function(request) {
 		}.bind(this));
 	}
 };
+module.exports.disableCaching = function () {
```
Comment (michael-ciniawsky) [inline loader.js]: Better implemented as a loader option (`{ cache: false }`)

**Comment by michael-ciniawsky** [comment]
Will be superseded by a new PR soon, closing due to inactivity. Thx 😛

**Pull request closed without merging**